docs/core/diagnostics/lldb-linux.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Install and use LLDB on Linux
+description: Instructions for installing and using LLDB to debug a .NET application on Linux
+ms.date: 02/13/2025
+---
+
+# Install and use LLDB on Linux
+
+[LLDB](https://lldb.llvm.org/) is a powerful, open-source debugger that's part of the LLVM project. When you're debugging .NET applications on Linux, you can use LLDB with the [.NET debugger extensions](debugger-extensions.md) to inspect managed application state. LLDB can work with both dumps and live processes.
+
+## Install
+
+### Install LLDB
+
+The .NET debugger extensions require at least LLDB 3.9 but version 10.0 or later is recommended. The following sections provide instructions for installing LLDB on popular Linux distributions.
+
+#### AzureLinux 2.0 and later
+
+To install the LLDB packages:
+
+```console
+    sudo tdnf install lldb
+```
+
+To launch LLDB:
+
+```console
+    lldb
+```
+
+#### Ubuntu 20.04 and later
+
+To install the LLDB packages:
+
+```console
+    sudo apt-get update
+    sudo apt-get install lldb
+```
+
+To launch LLDB:
+
+```console
+    lldb
+```
+
+#### Alpine 3.9 and later
+
+To install the LLDB packages:
+
+```console
+    apk update
+    apk add lldb
+```
+
+To launch LLDB:
+
+```console
+    lldb
+```
+
+#### Debian 9 and later
+
+To install the LLDB packages:
+
+```console
+    sudo apt-get install lldb-3.9 python-lldb-3.9
+```
+
+To launch LLDB:
+
+```console
+    lldb-3.9
+```
+
+#### Fedora 29 and later
+
+To install the LLDB packages:
+
+```console
+    sudo dnf install lldb python2-lldb
+```
+
+To launch LLDB:
+
+```console
+    lldb
+```
+
+#### RHEL 7.5 and later
+
+See [LLDB](https://access.redhat.com/documentation/en-us/red_hat_developer_tools/1/html/using_llvm_12.0.1_toolset/assembly_llvm#proc_installing-comp-toolset_assembly_llvm) on RedHat's website.
+
+### Install the .NET debugger extensions
+
+Install the .NET debugger extensions using the [dotnet-debugger-extensions](dotnet-debugger-extensions.md) install tool. The installer creates a `.lldbinit` file in your home directory that automatically loads the extensions when you start LLDB.
+
+> [!NOTE]
+> The version of the debugger extensions does not need to match the version of the .NET runtime. We recommend using the latest extension version.
+
+> [!NOTE]
+> Installing with [dotnet-sos](dotnet-sos.md) instead of [dotnet-debugger-extensions](dotnet-debugger-extensions.md) also works, but only installs a subset of the extension commands.
+
+## Getting started with LLDB
+
+This example shows using LLDB to attach to a pre-existing .NET application that is running on the machine.
+
+1. Use the `ps` command to find the process ID (PID) of the .NET application you want to debug.
+
+```console
+    $ ps -ao pid,cmd
+    PID CMD
+    697 -bash
+    229233 dotnet webapp.dll
+    229696 ps -ao pid,cmd
+    ... other processes omitted for brevity ...
+```
+
+In this case the PID of the .NET application we want to debug is 229233.
```

Comment:
````suggestion
   ```console
       $ ps -ao pid,cmd
       PID CMD
       697 -bash
       229233 dotnet webapp.dll
       229696 ps -ao pid,cmd
       ... other processes omitted for brevity ...
   ```

   In this case, the PID of the .NET application to debug is 229233.
````